includes/wf_crm_webform_base.inc

```diff
@@ -186,6 +190,36 @@ abstract class wf_crm_webform_base {
   }
 
   /**
+   * Reorder returned results according to settings chosen in wf_civicrm backend
+   *
+   * @param string $ent
+   * @param array $values
+   * @return array $reorderedArray
+   * @todo Handle user-select case
+   */
+  protected function reorderByLocationType($ent, $values = array()){
+    $reorderedArray = array();
+    // For every location field setting
+    foreach ($this->settings['data']['contact'] as $setting) {
```

Comment:
I think looping through every contact on the webform is wrong - should just focus on the one relevant contact.